homeassistant/components/hyundai_kia_connect/entity.py

```diff
@@ -0,0 +1,23 @@
+"""Base Entity for Hyundai / Kia Connect integration."""
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+
+from .const import BRANDS, DOMAIN, REGIONS
+
+
+class HyundaiKiaConnectEntity(CoordinatorEntity):
+    """Class for base entity for Hyundai / Kia Connect integration."""
+
+    def __init__(self, coordinator, vehicle):
+        """Initialize the base entity."""
+        super().__init__(coordinator)
+        self.vehicle = vehicle
+
+    @property
+    def device_info(self):
+        """Return device information to use for this entity."""
+        return {
+            "identifiers": {(DOMAIN, self.vehicle.id)},
+            "manufacturer": f"{BRANDS[self.coordinator.vehicle_manager.brand]} {REGIONS[self.coordinator.vehicle_manager.region]}",
+            "model": self.vehicle.model,
+            "name": self.vehicle.name,
+        }
```

Comment:
This should use the `DeviceInfo` class

https://github.com/home-assistant/core/blob/c0e9cfedfb584d00c525546285b16051c2b089c7/homeassistant/helpers/entity.py#L170

```suggestion
        return DeviceInfo(
            identifiers={(DOMAIN, self.vehicle.id)},
            manufacturer=f"{BRANDS[self.coordinator.vehicle_manager.brand]} {REGIONS[self.coordinator.vehicle_manager.region]}",
            model=self.vehicle.model,
            name=self.vehicle.name,
        )
```